src/main/java/com/rakovets/course/java/core/practice/oop_classes_and_objects/Time.java

```diff
@@ -0,0 +1,47 @@
+package com.rakovets.course.java.core.practice.oop_classes_and_objects;
+
+public class Time {
+    private int hours;
+    private int minutes;
+    private int seconds;
+
+    public Time(int hours, int minutes, int seconds) {
+        this.hours = hours;
+        this.minutes = minutes;
+        this.seconds = seconds;
+    }
+
+    public Time(long totalSeconds) {
+
+    }
+
+    public int getHours() {
+        return hours;
+    }
+
+    public void setHours(int hours) {
+        this.hours = hours;
+    }
+
+    public int getMinutes() {
+        return minutes;
+    }
+
+    public void setMinutes(int minutes) {
+        this.minutes = minutes;
+    }
+
+    public int getSeconds() {
+        return seconds;
+    }
+
+    public void setSeconds(int seconds) {
+        this.seconds = seconds;
+    }
```

Comment:
"Violation: Getters and setters should be placed at the very end
Solution: Move getters and setters to the end"